### DISCUSSION TITLE
Open Ai and Claude Ai not showing me answers in GPT MOBILE

### DISCUSSION THREAD

**Discussion by Nikhilsinha666** [Taewan-P/gpt_mobile#114] [Q&A]
Even though I pasted the API of Clude, OpenAI, and Gemini, only Gemini is showing me an answer, but not Claude or OpenAI. Please watch error screen recording uploaded to google drive
[https://drive.google.com/file/d/1yioo8dyBSFpX9D-goRVBSVulPdQozwLg/view?usp=drivesdk](url)

**Comment by Taewan-P** [comment]
Hi, please refer to #98. Both APIs use prepaid billing.
You can also see the message that the Claude API sent you.
